EIPS/eip-draft_reprice_opcodes.md

```diff
@@ -0,0 +1,145 @@
+---
+eip: <to be assigned>
+title: Opcode repricing 
```

Comment:
Could you make the title more descriptive as there might be more repricings in the future.

Reply:
done